etl/steps/data/garden/artificial_intelligence/2025-11-11/nvidia_revenue.py

```diff
@@ -0,0 +1,51 @@
+"""Garden step for NVIDIA quarterly revenue data."""
+
+from etl.helpers import PathFinder, create_dataset
```

Comment:
no need to import `create_dataset`, it is now a method of `PathFinder`


```suggestion
from etl.helpers import PathFinder
```